rand_os/src/sgx.rs

```diff
@@ -0,0 +1,30 @@
+use super::OsRngImpl;
```

Comment:
Can you include the licence boilerplate please? You only need the first copyright line ("Developers of the Rand project")

Reply:
I've included the license boilerplate.